### PR TITLE
Some tuning of the documentation index.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

**Review by ekpyron — DISMISSED** [review]

File: docs/index.rst [hunk 1]
```diff
@@ -34,7 +34,8 @@ crowdfunding, blind auctions, multi-signature wallets and more.
 Translations
 ------------
 
-This documentation is translated into several languages by community volunteers, but the English version stands as a reference.
+This documentation is translated into several languages by community volunteers
+with varying degrees of completeness and actuality. The English version stands as a reference.
```
Comment (ChrisChinchilla) [inline docs/index.rst]: I would say that especially in the context of this sentence, 'actuality' is overly complex, maybe 'accuracy' instead?

**Review by ChrisChinchilla — CHANGES_REQUESTED** [review]
One small thing I think needs changing.

File: docs/index.rst (hunk 1 above)
Comment (chriseth) [reply docs/index.rst]: I meant to say that some of them might be outdated.

Comment (chriseth) [reply docs/index.rst]: Yeah, that's not the right word in english ;)

**Comment by chriseth** [comment]
Updated.

File: docs/index.rst (hunk 1 above)
Comment (ChrisChinchilla) [reply docs/index.rst]: @chriseth I get you, "some might be outdated" sounds slightly rude, err, maybe just say "varying degrees of completeness", can't think of anything better right now after scouring the thesaurus. :)

**Comment by chriseth** [comment]
@ChrisChinchilla do you still would like to see changes here or is it fine as it is now?

**Review by ChrisChinchilla — APPROVED** [review]

**Pull request merged**